index.html

```diff
@@ -3,10 +3,155 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Document</title>
+  <link rel="stylesheet" href="/css/Nabvar.css">
+  <link rel="stylesheet" href="/css/burgerMenuWindows.css">
+  <link rel="stylesheet" href="/css/dinamicSection.css">
+  <title>CapstoneFernando</title>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 </head>
 <body>
-  
+  <div class="upVar">
+    <ul class="Uplist">
+      <a href=""><img src="/images/facebook.png" class="logFaceAndTwit" srcset=""></a>
+      <a href=""><img src="/images/twitter.png" class="logFaceAndTwit" srcset=""></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </ul>
+  </div>
+  <header class="header">
+    <div>
+      <nav class="mobile-navbar">
+        <button><img src="./images/mobile_menu 1.png" alt="burgerMenu" onclick ="opneNave();" class="image-burger-menu"></button>
+      </nav>
+    </div>
+    
+    <div class="container-logo">
+      <a href="index.html"><img src="./images/LogoUrban.png" alt= My Logo class="logo"></a>
+    </div>
+      <nav class="navbar">
+        <ul class="nav-list"> 
+          <li>
+            <a href="AboutPage.html">About</a>
+          </li>
+          <li>
+            <a href="#Program">Program</a>
+          </li>
```

Comment:
-  [ ] like the previous reviewer noted, kindly ensure that your nave items are even spaced and aligned like in the original design.
 
![Screenshot (1303)_LI](https://github.com/Fernando426/CapstoneN01/assets/65298048/fb7f8f19-9d1a-4fe8-a46b-bf922bf93531)